geometry.js

```diff
@@ -0,0 +1,20 @@
+function rect(width, height) {
+    return width * height;
+}
```

Comment:
Looking good! Its a good habit to add documentation for each method as well. 
Check out https://jsdoc.app/ and https://www.jetbrains.com/help/phpstorm/creating-jsdoc-comments.html

Reply:
Awesome! Thanks!